### ISSUE TITLE
Feedback on the Verify WZ Version option: setting "Fast" as the new default

### ISSUE THREAD

**Issue by PirateIzzy** [Kagamia/WzComparerR2#282]
So I was thinking that in the WCR2 settings, we could switch the Verify WZ Version's "Fast" and "Default" since most people using this tool for the first time may not know of this option, and as a result loading a MapleStory client will take a long time. 
![image](https://github.com/user-attachments/assets/9269d19a-0ae1-44ee-b6cb-e84171abb47d)

@HikariCalyx did this for his JMS build where he set Fast to the new default, and renamed the old Default to "Legacy". I think we can do the same upstream. https://github.com/HikariCalyx/WzComparerR2-JMS/commit/e0df652e1d28fcf7166e0e4084e9b8edebe00341#diff-3f9693f26feeef2feee7b023029b845a4cd4e658064bdd099ff992db3f937ea7R44 

Admittedly I'm not entirely sure of a scenario where someone would want to use "Default" as opposed to "Fast". I asked KENNYSOFT and he said it's to enable checking all checksums, but I'm wondering when someone would prefer that instead of the Fast verification. Is it for legacy WZ file versions (32-bit) or something else?

**Comment by Kagamia** [comment]
Yes, I think its time to set `fast strategy` as default, since it has been verified to be stable enough for over 2yrs.

**Comment by PirateIzzy** [comment]
Thank you. I've been pretty busy with the GMS branch, but I'll wait for the KMS branch to be updated first.
